kentik_api_library/kentik_api/kentik_api.py

```diff
@@ -24,8 +25,11 @@ class KentikAPI:
 
     API_VERSION = "v5"
 
-    def __init__(self, auth_email: str, auth_token: str, region: str = API_REGION_US) -> None:
-        if region.lower() == API_REGION_EU:
+    # TODO(dfurman): document function parameters
+    def __init__(self, auth_email: str, auth_token: str, region: str = API_REGION_US, api_url: str = None) -> None:
```

Comment:
I think we can simplify this constructor and provide just the api_url defaulted to API_URL_US

Reply:
Constructor simplidied